tool_path_planner/include/tool_path_planner/plane_slicer_raster_generator.h

```diff
@@ -126,7 +126,6 @@ class PlaneSlicerRasterGenerator : public PathGenerator
   };
 
   PlaneSlicerRasterGenerator() = default;
-  virtual ~PlaneSlicerRasterGenerator() = default;
```

Comment:
Could you explain the reasoning behind removing this line?

Reply:
Same as above; no need to define a destructor with no special behavior that doesn't properly override the base class destructor

Reply:
Ah, you caught me copying and pasting my comments because I couldn't think of something clever to say about the second instance of a particular change!  Woe is me!